prime-angular-frontend/src/app/shared/components/forms/email-form/email-form.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+
+import { FormUtilsService } from '@core/services/form-utils.service';
+import { ViewportService } from '@core/services/viewport.service';
+import { FormControlValidators } from '@lib/validators/form-control.validators';
+
+import { NextStepsFormState } from '@paper-enrolment/pages/next-steps-page/next-steps-form-state.class';
+@Component({
+  selector: 'app-email-form',
+  templateUrl: './email-form.component.html',
+  styleUrls: ['./email-form.component.scss']
+})
+export class EmailFormComponent implements OnInit {
```

Comment:
Wondering if this would be better named `EmailInputComponent` or `EmailFieldComponent`

Reply:
"EmailFormComponent" is consistent with other form components under the same structure.
![image](https://user-images.githubusercontent.com/108699279/199275592-ef5a32ff-d23d-467b-9c89-b717294d3a87.png)